src/website/app/demos/Button/examples/rtl.js

```diff
@@ -15,16 +15,20 @@
  */
 
 /* @flow */
+import ThemeProvder from '../../../../../ThemeProvider';
 import Button from '../../../../../Button';
 import IconBackspace from '../../../../../Icon/IconBackspace';
 
 export default {
   id: 'rtl',
   title: 'RTL Support',
-  description: '',
-  scope: { Button, IconBackspace },
+  description: `Buttons with Icons are reversed when rtl is set.
+A subset of Icons will automatically be reversed depending on their context.`,
```

Comment:
"A subset of Icons, those that convey directionality, will be reversed..."  - or similar